rust/connlib/clients/shared/src/messages.rs

```diff
@@ -95,11 +95,9 @@ mod test {
     use super::*;
     use chrono::DateTime;
     use connlib_shared::messages::{
-        DnsServer, Interface, IpDnsServer, Relay, ResourceDescription, ResourceDescriptionCidr,
-        ResourceDescriptionDns, Stun, Turn,
+        DnsServer, IpDnsServer, ResourceDescriptionCidr, ResourceDescriptionDns, Stun, Turn,
```

Comment:
I don't know why, but the unused deps lint triggered on some things here and in another chunk.